src/BudgetBadgerWebApi.Application/Mappings/Dtos/Category/CategoryDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BudgetBadgerWebApi.Application.Mappings.Dtos.Category
+{
+    public class CategoryDto
+    {
+        public int Id { get; set; }
+        public string Name { get; set; } = string.Empty;
```

Comment:
do we need default value here? This question corresponds to others places - if we have empty/null/default value on start, why do we need to set it at all? It could just be unassigned on start
```suggestion
        public string Name { get; set; }
```

Reply:
I can go with that. I will update the PR with this change.

Reply:
Changed